core/src/main/scala/Manifest.scala

```diff
@@ -116,7 +110,9 @@ object Manifest {
     policy: Option[ExpirationPolicy] = None,
     trafficShift: Option[TrafficShift] = None,
     volumes: List[Volume] = List.empty,
-    ephemeralDisk: Option[Int] = None
+    ephemeralDisk: Option[Int] = None,
```

Comment:
Given we have `volumes` now do we still want to keep `ephemeralDisk`? Prob not this PR but a future one